PRIVACY.md

```diff
@@ -1,6 +1,6 @@
 ## iD Editor Privacy Policy
 
-iD Editor is an open source OpenStreetMap editor. You are not required to use iD in order to use or edit OpenStreetMap, and the OpenStreetMap Foundation does not operate iD. You can learn more about OpenStreetMap and the OpenStreetMap Foundation at [https://www.openstreetmap.org/about](https://www.openstreetmap.org/about) and [https://wiki.osmfoundation.org/wiki/Main_Page](https://wiki.osmfoundation.org/wiki/Main_Page). You can learn more about the iD Editor project and its maintainers at [https://github.com/openstreetmap/iD](https://github.com/openstreetmap/iD).
+iD Editor is an open source OpenStreetMap editor. You are not required to use iD in order to use or edit OpenStreetMap. You can learn more about OpenStreetMap and the OpenStreetMap Foundation at [https://www.openstreetmap.org/about](https://www.openstreetmap.org/about) and [https://wiki.osmfoundation.org/wiki/Main_Page](https://wiki.osmfoundation.org/wiki/Main_Page). You can learn more about the iD Editor project and its maintainers at [https://github.com/openstreetmap/iD](https://github.com/openstreetmap/iD).
```

Comment:
The legal responsibility for the instance of iD the user is running lies with the operator of the website that is embedding the code (see comment from me on the PR). With other words for the iD instance from openstreetmap.org with the OSMF, for RapiD with Facebook and so on. With other words this text should probably be configurable per installation/build.